apps/viewer/src/hooks/usePlaybackControl.ts

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+
+import { ViewerVideoViewProps } from '../components/viewer-video-view';
+
+const usePlaybackControl = (sourceIds: string[]) => {
```

Comment:
What's the purpose of this hook? Could you please put the description of the function of this hook here?

Reply:
I've added a comment describing this hook.

It's used to manage playback control states (e.g. audio/video enabled, volume level) on a per-source basis. This needs to be done at the app-level so that state persists between unproject/reprojecting to and from the main stream. I've abstracted it into this hook so that the app file doesn't get too big.